classes/Uploader.php

```diff
@@ -63,6 +63,11 @@ public function getFilePath($file_name = null)
     {
         if (!isset($file_name)) {
             return tempnam($this->getSavePath(), $this->getUniqueFileName());
+        } else {
```

Comment:
It return a random file name when the $file_name var isn't set so we don't need to check the extension anyway, please advise